pkg/kubelet/api/v1alpha1/stats/types.go

```diff
@@ -133,6 +133,9 @@ type MemoryStats struct {
 	// The amount of working set memory. This includes recently accessed memory,
 	// dirty memory, and kernel memory. UsageBytes is <= TotalBytes.
 	WorkingSetBytes *uint64 `json:"workingSetBytes,omitempty"`
+	// The amount of anonymous and swap cache memory (includes transparent
+	// hugepages).
+	RSSBytes *uint64 `json:"rssBytes, omitempty"`
```

Comment:
(nit) no space after `,` (not sure if it matters, but it's always written like this)